src/main/resources/templates/email/feedbackToInterviewee.txt

```diff
@@ -0,0 +1,26 @@
+Thank your for practicing your interviewing skills using InterviewMe on {{formatted_date}}. Your interviewer has submitted feedback for your interview which you can view below. 
+
+The values 1-10 represent whether your interviewer strongly disagreed or aggreed with the prompt. 
```

Comment:
Nit: "strongly disagreed(1) or agreed(10)" just to be clear

Reply:
Done